backend/force-ssl.spec.js

```diff
@@ -0,0 +1,50 @@
+jest.mock('./config');
+const config = require('./config');
+const forceSsl = require('./force-ssl');
+
+const mockNext = jest.fn(() => {
+});
+const mockRes = {
+  redirect: jest.fn(() => {
+  })
+};
+
+test('forcessl function exists', () => {
+  expect(forceSsl);
+  expect((typeof forceSsl)).toBe("function");
```

Comment:
So far everything looks good but this test looks redundant - the other tests will fail anyway if `forceSsl` is not a function.

Reply:
Good point! It was just a "Hello Jest!" ;)